jujugui/static/gui/src/app/app.js

```diff
@@ -1872,9 +1872,6 @@ YUI.add('juju-gui', function(Y) {
     */
     switchEnv: function(socketUrl, username, password) {
       console.log('switching to new socket URL:', socketUrl);
-      if (this.get('sandbox')) {
```

Comment:
It is supported, if you point your sandbox at a real JEM (as specified by the QA instructions).